It is not possible to compile the TSCH_MAC for Openmote-cc2538

I can't compile the example using TSCH for MAC layer.
- Target platform: openmote-cc2538
- Assuming a complete and functional toolchain running on Linux (Ubuntu 16.04)
- Assuming that you are at the contiki-ng root directory
- Code version: 
```
git show
commit b4a23467fadadcb613bf2dcea790f3939fddc2ff
Merge: c6b795d d85cece
```

1. Navigate to `/examples/hello-world`
2. Include the code `MAKE_MAC = MAKE_MAC_TSCH` to the Project Makefile
3. Run the `make` command: `make TARGET=openmote-cc2538 hello-world`

Expected result: successful compilation

Current result: recipe for target 'obj_openmote-cc2538/tsch-slot-operation.o' failed 

Error log: [contiki-bg-tsch-openmote-cc2538-errorlog.txt](https://github.com/contiki-ng/contiki-ng/files/2480414/contiki-bg-tsch-openmote-cc2538-errorlog.txt)

--- a/arch/cpu/cc2538/cc2538-def.h
+++ b/arch/cpu/cc2538/cc2538-def.h
@@ -36,11 +36,13 @@
 /*---------------------------------------------------------------------------*/
 #define RTIMER_ARCH_SECOND 32768
 /*---------------------------------------------------------------------------*/
+#define CC2538_PHY_OVERHEAD          3
+#define CC2538_BYTE_AIR_TIME         32
 /* 352us from calling transmit() until the SFD byte has been sent */
-#define CC2538_DELAY_BEFORE_TX     ((unsigned)US_TO_RTIMERTICKS(352))
+#define CC2538_DELAY_BEFORE_TX       ((unsigned)US_TO_RTIMERTICKS(352))
 /* 192us as in datasheet but ACKs are not always received, so adjusted to 250us */
-#define CC2538_DELAY_BEFORE_RX     ((unsigned)US_TO_RTIMERTICKS(250))
-#define CC2538_DELAY_BEFORE_DETECT 0
+#define CC2538_DELAY_BEFORE_RX       ((unsigned)US_TO_RTIMERTICKS(250))
+#define CC2538_DELAY_BEFORE_DETECT   0
 /* Frame filtering done in software */
 #define TSCH_CONF_HW_FRAME_FILTERING  0
 

--- a/arch/platform/cc2538dk/dev/board.h
+++ b/arch/platform/cc2538dk/dev/board.h
@@ -221,6 +221,18 @@
 /** @} */
 /*---------------------------------------------------------------------------*/
 /**
+ * \name CC2538 TSCH configuration
+ *
+ * @{
+ */
+#define RADIO_PHY_OVERHEAD        CC2538_PHY_OVERHEAD
+#define RADIO_BYTE_AIR_TIME       CC2538_BYTE_AIR_TIME
+#define RADIO_DELAY_BEFORE_TX     CC2538_DELAY_BEFORE_TX
+#define RADIO_DELAY_BEFORE_RX     CC2538_DELAY_BEFORE_RX
+#define RADIO_DELAY_BEFORE_DETECT CC2538_DELAY_BEFORE_DETECT
+/** @} */
+/*---------------------------------------------------------------------------*/
+/**
  * \name Device string used on startup
  * @{
  */

--- a/arch/platform/openmote-cc2538/board.h
+++ b/arch/platform/openmote-cc2538/board.h
@@ -175,6 +175,18 @@
 /** @} */
 /*---------------------------------------------------------------------------*/
 /**
+ * \name CC2538 TSCH configuration
+ *
+ * @{
+ */
+#define RADIO_PHY_OVERHEAD        CC2538_PHY_OVERHEAD
+#define RADIO_BYTE_AIR_TIME       CC2538_BYTE_AIR_TIME
+#define RADIO_DELAY_BEFORE_TX     CC2538_DELAY_BEFORE_TX
+#define RADIO_DELAY_BEFORE_RX     CC2538_DELAY_BEFORE_RX
+#define RADIO_DELAY_BEFORE_DETECT CC2538_DELAY_BEFORE_DETECT
+/** @} */
+/*---------------------------------------------------------------------------*/
+/**
  * \name Device string used on startup
  * @{
  */

--- a/tests/01-compile-base/Makefile
+++ b/tests/01-compile-base/Makefile
@@ -24,6 +24,8 @@ rpl-border-router/sky \
 slip-radio/sky \
 libs/ipv6-hooks/sky \
 nullnet/native \
+nullnet/sky \
+nullnet/sky:MAKE_MAC=MAKE_MAC_TSCH \
 mqtt-client/native \
 coap/coap-example-client/native \
 coap/coap-example-server/native \

--- a/tests/02-compile-arm-ports-01/Makefile
+++ b/tests/02-compile-arm-ports-01/Makefile
@@ -89,6 +89,8 @@ platform-specific/cc2538-common/pka/cc2538dk \
 hello-world/cc2538dk \
 rpl-border-router/cc2538dk \
 rpl-border-router/cc2538dk:MAKE_ROUTING=MAKE_ROUTING_RPL_CLASSIC \
+6tisch/simple-node/cc2538dk \
+6tisch/simple-node/cc2538dk:MAKE_WITH_SECURITY=1,MAKE_WITH_ORCHESTRA=1 \
 hello-world/nrf52dk \
 platform-specific/nrf52dk/coap-demo/coap-server/nrf52dk \
 platform-specific/nrf52dk/coap-demo/coap-client/nrf52dk:SERVER_IPV6_EP=ffff \

--- a/tests/03-compile-arm-ports-02/Makefile
+++ b/tests/03-compile-arm-ports-02/Makefile
@@ -68,6 +68,8 @@ libs/ipv6-hooks/openmote-cc2538 \
 libs/shell/openmote-cc2538 \
 libs/simple-energest/openmote-cc2538 \
 libs/deployment/openmote-cc2538 \
+6tisch/simple-node/openmote-cc2538 \
+6tisch/simple-node/openmote-cc2538:MAKE_WITH_SECURITY=1,MAKE_WITH_ORCHESTRA=1 \
 
 TOOLS=
 

